### PR TITLE
Remove @dialonce/boot module

### PR DESCRIPTION
**Pull request by ykhatal**
I removed the `@dialonce/boot` because:
- Since bunnymq is an opensource module, we should let the user choose its transport
- It was causing an issue when i tested it on one of our projects

### CODE REVIEW

**Review by Spring3 — APPROVED** [review]

**Review by jkernech — CHANGES_REQUESTED** [review]
Can you add some details about the issues with the service-devices ? Also tests should be fixed

**Comment by ykhatal** [comment]
@jkernech test are fixed now, the issue is about the `bugsnag` not being initialized.

File: src/modules/retrocompat-config.js
Comment (dialonce) [inline src/modules/retrocompat-config.js]: MAJOR Merge this if statement with the nested one. [rule](https://sonar.dialonce.io/coding_rules#rule_key=javascript%3ACollapsibleIfStatements)

**Comment by dialonce** [comment]
SonarQube analysis reported 1 issue
* ![MAJOR][MAJOR] 1 major

Watch the comments in this conversation to review them.

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Review by mrister — APPROVED** [review]

File: src/modules/retrocompat-config.js
Comment (ykhatal) [inline src/modules/retrocompat-config.js]: @dial-once/developers sonar issue will be fixed shortly

**Review by jkernech — APPROVED** [review]
Fix Sonar issue and we are good to go !

**Comment by ykhatal** [comment]
@williamluang @ofkindness sorry guys, but i will not wait for review since you're out for lunch and i need to complete this before i leave

**Pull request merged**
